temp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
No, no. I mean keep this folders, but create special ignore rules to main .gitignore. ;-)

```
!/temp
/temp/*.*
```